### PR TITLE
feat: python script to set settings of an index

### PR DESCRIPTION
**Pull request by ElsaLebas**
Add python script example to set settings of an index

**Contributing to Algolia Samples**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: python/set_settings.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ALGOLIA_API_KEY = os.getenv('ALGOLIA_API_KEY')
+ALGOLIA_INDEX_NAME = os.getenv('ALGOLIA_INDEX_NAME')
+
+# Initialize the Algolia Client
+client = SearchClient.create(ALGOLIA_APP_ID, ALGOLIA_API_KEY)
+
+# Initialize an index
+index = client.init_index(ALGOLIA_INDEX_NAME)
+
+# Example of a record structure
```
Comment (matthewbond) [inline python/set_settings.py:20]: is this needed?

File: python/set_settings.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  # "objectID": 1,
+  # "name": "John Doe",
+  # "address": "London",
+  # "followers": 5
+  # }
+
+# Set your index settings 
+index.set_settings({
+  "searchableAttributes": ["name", "address"],
+  'attributesForFaceting': ["name", "filterOnly(address)","searchable(followers)"],
```
Comment (matthewbond) [inline python/set_settings.py:31]: maybe this is a good place to briefly explain why we are using filter only and searchable? 
not necessary but nice to have
https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/

File: python/set_settings.py (hunk 1 above)
Comment (ElsaLebas) [reply python/set_settings.py:20]: I think it is if we want to be able to interact with the ALGOLIA_INDEX_NAME index I think we need to first init it 
cf [docs #creating Index ](https://www.algolia.com/doc/api-client/methods/indexing/#creating-indices): _`initIndex` instantiates an object that allows you to interact with an Algolia index at the API level_
but I am not sure ...

Comment (matthewbond) [reply python/set_settings.py:20]: yes you are correct, sorry I was referring to:

`# Example of a record structure 
# record = {
  # "objectID": 1,
  # "name": "John Doe",
  # "address": "London",
  # "followers": 5
  # }`

**Review by clemfromspace — APPROVED** [review]

**Comment by matthewbond** [comment]
@ElsaLebas ready for you to merge! 🎉

**Pull request merged**